pkg/awsutils/awsutils.go

```diff
@@ -700,7 +700,9 @@ func (cache *EC2InstanceMetadataCache) createENI(useCustomCfg bool, sg []*string
 
 	if useCustomCfg {
 		log.Info("Using a custom network config for the new ENI")
-		input.Groups = sg
+		if len(sg) != 0 {
+			input.Groups = sg
+		}
```

Comment:
I think we should log a warning (or maybe just Info?) here, something like:
```
} else {
	log.Warnf("No custom networking security group found, will use same as the node: %s", input.Groups)
}
```

Reply:
Yeah, I initially thought of adding but since there was a log below which lists the SGs so I didn't do it. Will add a warning, it will be more clear when logs are checked. Thank you.